src/librepgp/stream-packet.cpp

```diff
@@ -1077,6 +1077,23 @@ stream_write_signature(const pgp_signature_t *sig, pgp_dest_t *dst)
     return false;
 }
 
+rnp_result_t
+stream_parse_marker(pgp_source_t &src)
+{
+    pgp_packet_body_t pkt = {};
+    rnp_result_t      res = stream_read_packet_body(&src, &pkt);
+    if (res) {
+        return res;
+    }
+
+    if ((pkt.len != 3) || memcmp(pkt.data, "PGP", 3)) {
```

Comment:
I suggest to have a marker constant "PGP" and re-use it.

Reply:
Thanks, will update.